api/README.md

```diff
@@ -62,12 +123,34 @@ __`filter`__ : Differences to include in the summary.
 
 Performs an HTTP redirect for the results summary JSON blob of the given TestRun.
 
+__Response format__
+
+The summary JSON is in the format
+
+    {
+      "/path/to/test.html": [1, 1],
+    }
+
+Where the array contains [`number of passes`, `total tests`].
+
 __Parameters__
 
-__`browser`__ : Browser to fetch the results for, e.g. `chrome`
+__`product`__ : Product to fetch the results for, e.g. `chrome-66`
```

Comment:
This one has a version specifier, but previous `product` examples just had an unversioned `chrome`. How do I know what "products" exist, and what I should be asking for?

Reply:
We don't (currently) have a way to fetch "all the products".

FWIW, your use case the products (unversioned) are what you'd want to call with, i.e.

- `chrome`
- `edge`
- `firefox`
- `safari`

Since it's ordered by date desc, that's the latest version of each.